content/docs/event_loop.mdz

```diff
@@ -3,168 +3,217 @@
+not use them in applications that use the event loop. These functions may
+be changed or deprecated in the future. Other blocking functions include
+@code`os/sleep` and @code`getline`.
+
+NOTE: This means that getting input from the repl will block the event
+loop! Be careful of this when trying to use ev functions from the built-in
+repl. Instead, you can use a stream-based repl such as @code`spork/netrepl`,
+which will interact well with the event loop.
 
 ## Creating Tasks
 
-A default Janet program has a single task that will run until complete. To create new tasks, Janet provides two
-built-in functions - @code`ev/go` and @code`ev/call`. @code`ev/call` can be implement in terms of @code`ev/go`, and other
-macros and functions are wrappers around these two functions.
+A default Janet program has a single task that will run until complete. To
+create new tasks, Janet provides two built-in functions - @code`ev/go` and
+@code`ev/call`. @code`ev/call` can be implement in terms of @code`ev/go`,
+and other macros and functions are wrappers around these two functions.
 
 @codeblock[janet]```
 (defn worker
- "Does some work."
- [name n]
- (for i 0 n
-  (print name " working " i "...")
-  (ev/sleep 0.5))
- (print name " is done!"))
-
-# Start bob working in a new task with ev/call
+  "Does some work."
+  [name n]
+  (for i 0 n
+    (print name " working " i "...")
+    (ev/sleep 0.5))
+  (print name " is done!"))
+
+# Start bob working on a new task with ev/call
 (ev/call worker "bob" 10)
 
 (ev/sleep 0.25)
 
-# Start sally working in a new task with ev/go
+# Start sally working on a new task with ev/go
 (ev/go (fiber/new |(worker "sally" 20)))
 
 (ev/sleep 11)
 (print "Everyone should be done by now!")
+
 ```
 
-This example will start two worker tasks that will run concurrently with each other. The @code`ev/call` function takes a function as well as it's arguments
-and calls that function inside a new task. @code`ev/call` returns the newly created task fiber immediately.
+This example will start two worker tasks that will run concurrently with
+each other. The @code`ev/call` function takes a function as well as it's
+arguments and calls that function inside a new task. @code`ev/call` returns
+the newly created task fiber immediately.
 
-The more general way to start new tasks is @code`ev/go`, which takes as an argument a fiber to be resumed by the event loop scheduler.
-@code`ev/go` allows the programmer to customize fiber flags as well as specify a supervisor channel for the newly created task.
+The more general way to start new tasks is @code`ev/go`, which takes as an
+argument a fiber to be resumed by the event loop scheduler.
 
-Lastly, there is the @code`ev/spawn` macro for concisely running a series of forms in a new task.
+The @code`ev/go` allows the programmer to customize fiber flags and specify a
+supervisor channel for the newly created task.
+
+Last is the @code`ev/spawn` macro for concisely running a series of forms
```

Comment:
This sounds unnatural.

Reply:
If you have any recommendations, that would be awesome.